style.css

```diff
@@ -34,11 +35,93 @@ html {
   font-family: var(--font-secondary);
   font-weight: 300;
   font-size: 24px;
-  color: var(--gray-dark);
 }
 
- /*@media (max-width: 1024px) {
-}*/
+.hero {
+  margin-top: 20px;
+}
+.hero__content {
+  display: flex;
+  gap: 60px;
+}
+.hero__welcome {
+  font-family: var(--font-secondary);
+  font-size: 34px;
+  display: flex;
+  align-items: flex-end;
+  margin-top: 0;
+}
+.hero__welcome::before {
+  content: "";
+  height: 0.5em;
+  width: 48px;
+  border-top: 2px solid var(--gray-dark);
+  margin-right: 12px;
+}
+.hero__title {
+  margin-top: 20px;
```

Comment:
Please use one way margins. Fix this in the other places too. It's best to remove all default margins so you can put them  yourself only were you need them.